ceph_deploy/rgw.py

```diff
@@ -174,13 +173,13 @@ def rgw_create(args):
                 msg = "exisiting rgw '%s:%s' has a different hostname" % (hostname, name)
                 LOG.error(msg)
                 raise RuntimeError(msg)
-
+        rgw_dns_name_value = "%s:%s" % (hostname, port)
         if cfg.has_option(enitity,'rgw_dns_name') is False:
-            cfg.set(enitity, 'rgw_dns_name', hostname)
+            cfg.set(enitity, 'rgw_dns_name', rgw_dns_name_value)
             changed_cfg = True
         else:
             existing_value = cfg.get(enitity, 'rgw_dns_name')
```

Comment:
BTW is this var named `enitity` everywhere else also, then its ok.. otherwise might be a type